stories/index.js

```diff
@@ -666,6 +664,69 @@ storiesOf('TableHeading', module)
 
 storiesOf('Table', module)
   .add('base table', () => {
+    const noResults = props => (
+      <p>Nothing!</p>
+    );
+
+    return (
+      <Table
+        NoResults={noResults}
+      />
+    );
+  })
+
+  .add('empty with columns', () => {
+    const components = {
+      Table: ({ TableHeading, TableBody, NoResults, style, visibleRows }) => (
+        <table style={style}>
+          <TableHeading />
+          { visibleRows ? (TableBody && <TableBody />) : (NoResults && <NoResults />) }
+        </table>
+      ),
+      NoResultsContainer: compose(
```

Comment:
I may be wrong, but my understanding of `compose()` is that it's already returning a higher-order component. The outer arrow function seems to be redundant; `compose` is already returning a function that takes in the original component and returns the composed component.